teps/0002-custom-tasks.md

```diff
@@ -414,3 +414,13 @@ None.
 ## Upgrade & Migration Strategy (optional)
 
 TBD. At this time, the proposal only covers adding new a type and documentating the contract. If changes to the types or contract are deemed necessary in the future, in response to feedback, then an upgrade/migration strategy might be necessary.
+
+# Open Questions
+
+* Should Tekton's controller be responsible for updating `Run`s' `.status.conditions` in the case of cancellation and timeout (as it does when enforcing [initial update timeout](#initial-update-timeout)), or should these updates be the sole responsibility of Custom Task authors?
+
+* Package name and helper methods included in `tektoncd/pipeline` repo to aid Custom Task authors writing their controllers in Go; and should we expect them to use `knative/pkg` helpers too?
```

Comment:
i dont think we should require they use any particular helper, but i agree that it makes sense for us to provide examples and docs

Reply:
maybe a segue but i also think it would be very handy to users if we had an example repo that shows how to setup tekton in general, so if you wanted to get started you could copy the repo and go from there. maybe could be the same repo?

Reply:
+1, we shouldn't _require_ anything -- you should be able to write a controller in bash if you were really dedicated/insane -- but I think we can be opinionated in providing an easy getting-started guide for the vast majority of integrators who just want to build stuff and don't care exactly how.